test/integration/scene_broadcaster_system.cc

```diff
@@ -914,6 +914,44 @@ TEST_P(SceneBroadcasterTest,
   server.Run(true, 1, false);
 }
 
+/////////////////////////////////////////////////
+TEST_P(SceneBroadcasterTest,
+    IGN_UTILS_TEST_DISABLED_ON_WIN32(SceneInfoHasSceneSdf))
+{
+  // Start server
+  ignition::gazebo::ServerConfig serverConfig;
+  serverConfig.SetSdfFile(std::string(PROJECT_SOURCE_PATH) +
+      "/test/worlds/conveyor.sdf");
```

Comment:
`jointPaths`

Reply:
Fixed in [db49ac7](https://github.com/gazebosim/gz-sim/pull/1508/commits/db49ac7bddd625cec15414dcf51b24eaf31bf3db)